packages/std/src/results/cosmos_msg.rs

```diff
@@ -133,7 +134,9 @@ pub enum WasmMsg {
     },
 }
 
-/// Shortcut helper as the construction of WasmMsg::Instantiate can be quite verbose in contract code
+/// Shortcut helper as the construction of WasmMsg::Instantiate can be quite verbose in contract code.
+///
+/// When using this, `admin` is always unset. If you need more flexibility, create the message directly.
```

Comment:
That is a nice way to avoid breaking existing code.

Maybe add another method `wasm_instantiate_with_admin`?

Reply:
I think if you do complex stuff in contracts, you can affort 8 lines of code to create the exact message you want to have. If your contract needs it often, you can create your helper.